siriuspy/siriuspy/devices/lillrf.py

```diff
@@ -40,13 +42,31 @@ def amplitude(self, value):
 
     @property
     def phase(self):
-        """."""
+        """Phase."""
         return self['GET_PHASE']
 
     @phase.setter
     def phase(self, value):
         self['SET_PHASE'] = value
 
+    @property
+    def integral(self):
+        """Integral Enable."""
+        return self['GET_INTEGRAL_ENABLE']
+
+    @integral.setter
+    def integral(self, value):
+        self['SET_INTEGRAL_ENABLE'] = value
+
+    @property
+    def feedback(self):
```

Comment:
I think it is better to use `feedback_state`. What do you think?

Reply:
I agree!

Reply:
done!